src/servers/sapphire_zone/Network/Handlers/GMCommandHandlers.cpp

```diff
@@ -467,10 +467,11 @@ void Core::Network::GameConnection::gm1Handler( const Packets::GamePacket& inPac
       auto pCurrentZone = player.getCurrentZone();
       player.sendNotice( "ZoneId: " + std::to_string( player.getZoneId() ) + "\nName: " +
                          pCurrentZone->getName() + "\nInternalName: " +
-                         pCurrentZone->getInternalName() + "\nPopCount: " +
+                         pCurrentZone->getInternalName() + "\nGuId: " +
```

Comment:
would be nice if you could fix this so the label comes before the getter on each line

    "\nName: " + pCurrentZone->getName() +
    "\nInternalName: " + pCurrentZone->getInternalName() +
    "\nGuid: " + .... etc